plugin/storage/integration/kafka_test.go

```diff
@@ -91,6 +93,10 @@ func (s *KafkaIntegrationTestSuite) initialize(t *testing.T) {
 	}
 	options.InitFromViper(v)
 	traceStore := memory.NewStore()
+	_, err = builder.CreateConsumer(logger, metrics.NullFactory, traceStore, options)
+	require.Error(t, err, "invalid version `0`")
```

Comment:
`"invalid version `0`"` this string is non-functional. If you want to test that the error has this text then use ErrorEquals or ErrorContains